locales/es/messages.po

```diff
@@ -1654,6 +1658,10 @@ msgstr "Esta herramienta te la trae JustFix.nyc. Somos una organización sin fin
 msgid "To begin the password reset process, we'll text you a verification code."
 msgstr "Para restablecer tu contraseña, te enviaremos un código de verificación."
 
+#: frontend/lib/norent/letter-email-to-user.tsx:29
+msgid "To learn more about what to do next, check out our FAQ page: {faqURL}"
+msgstr "Para saber más sobre qué hacer a continuación, consulta nuestra página de preguntas más frecuentes: {faqURL}"
+
```

Comment:
FYI @toolness I copied over the existing Spanish translation here from before. Once this is merged, I'll be sure to update CrowdIn too so that it all stays consistent.